internal/runtime/config.go

```diff
@@ -74,6 +75,7 @@ func (c *Config) storeContainerPolicyConfiguration(vcfg viper.Viper) {
 	c.Platform = vcfg.GetString("platform")
 	c.Insecure = vcfg.GetBool("insecure")
 	c.Offline = vcfg.GetBool("offline")
+	c.Konflux = vcfg.GetBool("konflux")
```

Comment:
Is this enough to get set? Setting `PFLT_KONFLUX=1` in the environment enables this?

Reply:
Yes, `PFLT_KONFLUX=1` or `PFLT_KONFLUX=true` are all that's needed for viper to do it's magic.